benchmarks/views/user.py

```diff
@@ -198,6 +204,40 @@ def post(self, request):
         return render(request, 'benchmarks/success.html', {"domain": self.domain})
 
 
+def is_submission_original(file, submitter):
+    # add metrics and data eventually
+    plugin_db_mapping = {"models": Model, "benchmarks": BenchmarkType}
+
+    with zipfile.ZipFile(file, mode="r") as archive:
+        namelist = archive.infolist()
+        plugins = plugins_exist(namelist)[1]
+
+        # for each plugin submitted, make sure that the identifier does not exist already:
+        for plugin in plugins:
+            identifiers = plugin_has_instances(namelist, plugin)[1]
+            db_table = plugin_db_mapping[plugin]
+
+            # Determine the field name based on the plugin type
+            field_name = 'name' if plugin == "models" else 'identifier'
+
+            for identifier in identifiers:
+                query_filter = {field_name: identifier}
+
+                # Check if an entry with the given identifier exists
+                if db_table.objects.filter(**query_filter).exists():
+                    owner_obj = db_table.objects.get(**query_filter)
+                    owner_id = getattr(owner_obj, 'owner_id', None) or getattr(owner_obj, 'owner').id
+
+                    # Check to see if the submitter is the owner (or superuser)
+                    if owner_id == submitter.id or submitter.is_superuser:
+                        # Khaled versioning here
+                        print(owner_id, submitter)
+                    else:
+                        return False, [plugin, identifier]
```

Comment:
```suggestion
                    return False, [plugin, identifier]
```
can add back with the versioning PR
(btw, you should be able to write `owner == submitter`, no need for ids)